### ISSUE TITLE
Bug: "Great Slave Lake"

### ISSUE THREAD

**Issue by lintool** [gwf-uwaterloo/waterwheel#20]
```
>>> doc = nlp('The Mackenzie River flows from the Great Slave Lake into the Arctic Ocean.')
>>> for ent in doc.ents:
...     print(f'{ent.text}, type: {ent.label_} [{ent.start_char}, {ent.end_char}]')
... 
Mackenzie River, type: RIVER [4, 19]
Slave, type: RIVER [41, 46]
Arctic, type: LAKE [61, 67]
```
Great Slave Lake is here: https://www.wikidata.org/wiki/Q5539

ref: #19 - will address "Arctic Ocean" issue.

**Comment by lintool** [comment]
@Govind9 

```
>>> import spacy
>>> from waterwheel import WaterWheel
>>> 
>>> nlp = spacy.load('en_core_web_sm')
>>> nlp.add_pipe(WaterWheel(nlp))
>>> doc = nlp('The Mackenzie River flows from the Great Slave Lake into the Arctic Ocean.')
>>> for ent in doc.ents:
...     print(f'{ent.text}, type: {ent.label_}, {ent._.wikilink} [{ent.start_char}, {ent.end_char}]')
... 
Mackenzie, type: RIVER, None [4, 13]
Great Slave, type: LAKE, None [35, 46]
Arctic, type: LAKE, None [61, 67]
```
`ent._.wikilink` shows `NONE`... so I think something broke?

**Comment by Govind9** [comment]
PL sent with fix.

**Comment by lintool** [comment]
Fixed with 4234c82
